src/server/hotspot_partition_data.h

```diff
@@ -4,21 +4,26 @@
 
 #pragma once
 
-#include "shell/commands.h"
+#include "shell/command_helper.h"
 
 namespace pegasus {
 namespace server {
 
+enum partition_qps_type
+{
+    READ_HOTSPOT_DATA = 0,
+    WRITE_HOTSPOT_DATA
```

Comment:
why `READ_HOTSPOT_DATA` init explictly, but `WRITE_HOTSPOT_DATA` no?

Reply:
in cpp, the enum is increased by one orderly. so only need to init the first one

Reply:
The first default 0, so also can not be inited?